rabbitinahat/src/main/java/org/ohdsi/rabbitInAHat/DetailsPanel.java

```diff
@@ -381,19 +381,15 @@ public void initialise() {
 			table.setBorder(new MatteBorder(1, 0, 1, 0, Color.BLACK));
 			table.setCellSelectionEnabled(true);
 
-			// Make first column wider if source panel or second column if this is a target field panel
-			int wideColumnIndex = isTargetFieldPanel ? 1 : 0;
-			TableColumn column;
-			for (int i = 0; i < table.getColumnCount(); i++) {
-				column = table.getColumnModel().getColumn(i);
-				if (i == wideColumnIndex) {
-					column.setPreferredWidth(500);
-				} else {
-					column.setPreferredWidth(50);
-				}
+			if (isTargetFieldPanel) {
+				// Wide columns for concept name and class id
+				table.getColumnModel().getColumn(1).setPreferredWidth(100);
+				table.getColumnModel().getColumn(2).setPreferredWidth(100);
 			}
 
 			if (!isTargetFieldPanel) {
+				// Wide column for value name
+				table.getColumnModel().getColumn(0).setPreferredWidth(500);

```

Comment:
In the figure, it looks like the concept name should get the most width, and the ID and type could get less. Is that possible to realise?